src/main/java/com/kodilla/ecommercee/domain/Product.java

```diff
@@ -1,9 +1,40 @@
 package com.kodilla.ecommercee.domain;
 
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.math.BigDecimal;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity(name = "PRODUCTS")
 public class Product {
+
+
+    @Id
+    @NotNull
+    @GeneratedValue
+    @Column(name = "PRODUCT_ID", unique = true)
     private Long id;
+
+    @Column(name = "NAME")
     private String name;
+
+    @Column(name = "DESCRIPTION")
     private String description;
-    private Double price;
-    private Long groupId;
+
+    @Column(name = "PRICE")
+    private BigDecimal price;
+
+    //@ManyToOne
+    //@Column(name = "GROUP_ID")
+    //private GroupId groupId;
```

Comment:
Komentarze nie są potrzebne. W repozytorium jest encja [Group](https://github.com/pplutap/project-jdp-2209-01/blob/main/src/main/java/com/kodilla/ecommercee/domain/Group.java)

Reply:
To nie mam tym bardziej pomysłu jak ogarnąć tą kolumne Group_Id, gdy private GroupId groupId podkreśla mi na czerwono i nie może się przez to zbudować projekt

Reply:
Czy w tym przypadku typ to będzie nazwa klasy jaką stworzyła ania, czyli po prostu "Group"?

Reply:
I w linii 34 powinno byc @JoinColumn , z tego powodu nie kompilowalo Ci aplikacji

Reply:
Ten typ Group już jakoś ogarnąłem, więc dziękuję za potwierdzenie i ta adnotacja @JoinColumn rzeczywiście może być rozwiązaniem. Sprawdzę i mam nadzieję, że ładnie się skompiluje